core/src/main/java/de/javakaffee/web/msm/BackupSessionTask.java

```diff
@@ -190,18 +186,15 @@ private void releaseLock() {
      *
      * @return the {@link BackupResultStatus}
      */
-    BackupResult doBackupSession( final MemcachedBackupSession session, final byte[] data, final byte[] attributesData ) throws InterruptedException {
+    BackupResult doBackupSession( final MemcachedBackupSession session, final byte[] data, final byte[] attributesData ) {
         if ( _log.isDebugEnabled() ) {
             _log.debug( "Trying to store session in memcached: " + session.getId() );
         }
 
         try {
             storeSessionInMemcached( session, data );
             return new BackupResult( BackupResultStatus.SUCCESS, data, attributesData );
-        } catch (final ExecutionException e) {
```

Comment:
This one wrapped all exceptions that were thrown by the future task - without having further context I'd say that `Exception` or `Throwable` should now be caught as a proper replacement.